### PR TITLE
Increase 2FA security

### PR DESCRIPTION
**Pull request by andrewoh531**
- 2FA code increased from 5 to 6 characters
- At any time, there is only 1 active 2FA code (down from from 10)
- Made failed password attempt count separate to failed 2FA verification count

### CODE REVIEW

File: api/app/config.py [hunk 1]
```diff
@@ -143,7 +143,8 @@ class Config(object):
     SMS_CHAR_COUNT_LIMIT = 495
     TEST_MESSAGE_FILENAME = 'Test message'
     ONE_OFF_MESSAGE_FILENAME = 'Report'
-    MAX_VERIFY_CODE_COUNT = 10
+    MAX_VERIFY_CODE_COUNT = 1
```
Comment (dagoof) [inline api/app/config.py]: does this get used anywhere anymore?

**Comment by dalehurwitz** [comment]
@andrewoh531 it would be nice to increase the size of the sms code input field

**Comment by jonathaningram** [comment]
@andrewoh531 got an error when trying to sign in. Steps I did were:

1. Create account
2. Sign out
3. Try and sign in again

Got error:

```
AttributeError: 'User' object has no attribute 'max_failed_verify_count'
  File "flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app/main/views/sign_in.py", line 28, in sign_in
    user = _get_and_verify_user(user, form.password.data)
  File "app/main/views/sign_in.py", line 91, in _get_and_verify_user
    elif user.is_locked():
  File "app/notify_client/models.py", line 148, in is_locked
    return self.failed_login_count >= self.max_failed_login_count or self.failed_verify_count >= self.max_failed_verify_count
```

See Sentry.

File: api/app/config.py (hunk 1 above)
Comment (andrewoh531) [reply api/app/config.py]: Yeah it's used [here](https://github.com/govau/notify/pull/185/files#diff-92235129d06e5131f2efeb1f4bed2e9bR142)

**Comment by andrewoh531** [comment]
@jonathaningram @dalehurwitz Changes applied

File: api/tests/app/user/test_rest_verify.py [hunk 2]
```diff
@@ -417,7 +416,7 @@ def test_send_email_code_returns_404_for_bad_input_data(admin_request):
 
 @freeze_time('2016-01-01T12:00:00')
 def test_user_verify_email_code(admin_request, sample_user):
-    magic_code = str(uuid.uuid4())
+    magic_code = str(uuid.uuid4()).upper()
```
Comment (jonathaningram) [inline api/tests/app/user/test_rest_verify.py]: @andrewoh531 I suppose you don't need this change and the line below anymore?

**Pull request merged**